mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -107,10 +107,14 @@ def test_acq_skip(tmp_path):
 
 def test_fix_types():
     """Test fixing of channel types."""
-    for fname, change in ((hp_fif_fname, True), (test_fif_fname, False),
-                          (ctf_fname, False)):
+    for fname, change, bads in (
+        (hp_fif_fname, True, ["MEG0111"]),
```

Comment:
Need to mark at least one channel as bad to see that its coil type also changed.